### ISSUE TITLE
jei fluid preview

### ISSUE THREAD

**Issue by ghost** [Low-Drag-MC/Multiblocked2#77]
adding a fluid to the recipe doesn't show in the jei preview:

![2024-12-11_15 21 38](https://github.com/user-attachments/assets/b3dc4554-1eaa-47e0-bfa2-9ce58e69d055)

![2024-12-11_15 21 34](https://github.com/user-attachments/assets/35dd5b94-2edb-447e-b8a2-7d3f8c74d7ae)

![2024-12-11_15 26 08](https://github.com/user-attachments/assets/a35f7a86-99d5-4cfe-af41-3e749ed24e11)

**Comment by Yefancy** [comment]
could you send me your mbd2 folder? btw, whats your jei version?

**Comment by ghost** [comment]
[mbd2.zip](https://github.com/user-attachments/files/18094635/mbd2.zip)

jei file name: "jei-1.20.1-forge-15.20.0.105"

**Comment by Yefancy** [comment]
![image](https://github.com/user-attachments/assets/10b9d73f-d0e0-4934-8511-582dc4a9b840)
its woking well with `jei-1.20.1-forge-15.20.0.105`

**Comment by Yefancy** [comment]
whats your ldlib version? are you using multiple recipe viewer?

**Comment by ghost** [comment]
ldlib-forge-1.20.1-1.0.33.b

what do you mean by multiple recipe viewer?

**Comment by Yefancy** [comment]
like, emi+jei, jei+rei

**Comment by ghost** [comment]
there are also weird overlays in jei that dont allow me to hover over said slot unless its from the side like this:

![2024-12-11_15 48 58](https://github.com/user-attachments/assets/42a5e252-176f-4696-9b4f-d900a52c5d7d)

**Comment by ghost** [comment]
no only jei

**Comment by Yefancy** [comment]
whats your gtm version

**Comment by ghost** [comment]
gtceu-1.20.1-1.5.4

**Comment by Yefancy** [comment]
it seems that incompatible with the gtm, lemme have a look

**Comment by ghost** [comment]
any result?

**Comment by Yefancy** [comment]
yeah, its an issue of gtm. but they are reworking ui stuff. have to wait for their updates to fix it

**Comment by ghost** [comment]
but it also doesnt work with vanilla fluids such as water

**Comment by Yefancy** [comment]
gtm is currently using the same ui system from ldlib. they modiied its behaviour. I have report it to the main dev of them

**Comment by austinsupercar** [comment]
I also am experiencing the same bug, I would appreciate it if you did fix this so I can use fluids in my recipes, this bug limits what's possible with the mod.

**Comment by ghost** [comment]
Any updates?

**Comment by Yefancy** [comment]
fixed since 1.0.22, but it requires gtm 1.6.4+, wait for their updates
